src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsregisterstodeliusupdate/services/CourtRegisterService.kt

```diff
@@ -11,6 +12,8 @@ import reactor.core.publisher.Mono
 @Service
 class CourtRegisterService(@Qualifier("courtRegisterApiWebClient") private val webClient: WebClient) {
 
+  private val courts = object : ParameterizedTypeReference<List<CourtDto>>() {}
```

Comment:
Very minor, but maybe name this `courtsType` as `courts` sounds like a `List<CourtDto>>`?

Reply:
Will update